snappiershot/snapshot/metadata.py

```diff
@@ -1,11 +1,48 @@
 """ Metadata object used to identity and track snapshots. """
-from typing import Dict
+from typing import Any, Dict
+
+from numpy import all, ndarray
 
 from ..inspection import CallerInfo
 
 
+def compare_metadata(from_test_function: Any, from_snapshot: Any) -> bool:
+    """Instantiate object from a dictionary read from file in order to compare to SnappierShot input.
+
+    SnappierShot serializes class objects by decomposing them into their parts in the form of a dictionary, then
+    writes to the snapshot JSON. Thus, SnappierShot must attempt to re-instantiate the dictionary form of the
+    objects when comparing inputs to a test function to those inputs serialized to JSON.
```

Comment:
suggest to reword for clarity: "... to a test function to the stored metadata inputs already serialized to snapshot JSON"

Reply:
fixed [e992527](https://github.com/MORSECorp/snappiershot/pull/96/commits/e992527d9d55b6e0c1505b7b76461e0b5c9df7e9)